jina/executors/indexers/cache.py

```diff
@@ -127,11 +127,13 @@ def delete(self, keys: Iterator['UniqueId'], *args, **kwargs):
 
     def get_add_handler(self):
         # not needed, as we use the queryhandler
-        pass
+        # FIXME better way to silence warnings
+        return 1
```

Comment:
if it is not used, raise a NotImplementedError

Reply:
This crashes the execution

Reply:
so why is this used? why does this even exist? I do prefer to have pass than return `1`